api/src/test/java/uk/gov/hmcts/fees2/register/api/controllers/FeeControllerTest.java

```diff
@@ -29,65 +29,29 @@
 
 public class FeeControllerTest extends BaseIntegrationTest {
 
-    private String feeCode;
-
     @Rule
     public ExpectedException thrown= ExpectedException.none();
 
-    /**
-     * @throws Exception
-     */
-    @Test
-    public synchronized void createFeeTest() throws Exception {
```

Comment:
Did you create a counterpart to this test with the new code-generation somewhere?

Reply:
Most of the tests are calling POST end-point for fee creation. And this test data is conflicting with others.